src/test/java/edu/cornell/kfs/vnd/batch/service/impl/CuCommodityCodeUpdateServiceImplTest.java

```diff
@@ -1,45 +1,56 @@
 package edu.cornell.kfs.vnd.batch.service.impl;
 
-import java.io.File;
-import java.util.ArrayList;
-import org.apache.commons.io.FileUtils;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertTrue;
 
-import org.kuali.kfs.sys.ConfigureContext;
-import org.kuali.kfs.sys.context.KualiTestBase;
-import org.kuali.kfs.sys.context.SpringContext;
+import java.util.ArrayList;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
 import org.kuali.kfs.vnd.businessobject.CommodityCode;
-import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
```

Comment:
Do you need to use PowerMockito for this test? Since the mocked service implementation appears to be in the same package as this test, and since the "updateCommodityCodes" method has "protected" visibility, you should be able to just use plain Mockito to create the spy and mock the protected method.

Reply:
Yes, I removed powermockito and was able to use Mockito with slightly different mocking syntax